### PR TITLE
Cleanup corelib page

### PR DESCRIPTION
**Pull request by mit-mit**
Clearly separate multi-platform libraries from platform specific libraries

Staged at https://mit-staging.firebaseapp.com/guides/libraries

### CODE REVIEW

File: src/_guides/libraries/index.md [hunk 1]
```diff
@@ -10,33 +10,58 @@ programming tasks such as working on collections of objects
...
-| [`dart:typed_data`][dart-typed_data]    <br> Lists that efficiently handle fixed sized data (for example, unsigned 8-byte integers) and SIMD numeric types. | All |
-| [`dart:web_audio`][dart-web_audio]      <br> High-fidelity audio programming in the browser. | Web |
-| [`dart:web_gl`][dart-web_gl]            <br> 3D programming in the browser. | Web 
-| [`dart:web_sql`][dart-web_sql]          <br> API for storing data in the browser that can be queried with SQL. | Web (obsolete) |
+| [`dart:async`][dart-async]              <br> Support for asynchronous programming, with classes such as Future and Stream. | |
+| [`dart:collection`][dart-collection]    <br> Classes and utilities that supplement the collection support in `dart:core`. | |
+| [`dart:convert`][dart-convert]          <br> Encoders and decoders for converting between different data representations, including JSON and UTF-8. | |
+| [`dart:core`][dart-core]                <br> Built-in types, collections, and other core functionality for every Dart program. | |
+| [`dart:developer`][dart-developer]      <br> Interaction with developer tools such as the debugger and inspector. | Experimental<br>JIT and dartdevc only) |
```
Comment (chalin) [inline src/_guides/libraries/index.md]: Opening parenthesis missing in the note.

**Comment by mit-mit** [comment]
Re-staged at https://mit-staging.firebaseapp.com/guides/libraries with recent edits

**Comment by mit-mit** [comment]
I don't love that the notes column across the three tables isn't the same width. Any ideas for how to do that?

File: src/_guides/libraries/index.md [hunk 2]
```diff
@@ -10,33 +9,60 @@ programming tasks such as working on collections of objects
...
+|-----------------------------------------------+-------------------------------|
+| Library                                       | Notes   |
+|-----------------------------------------------|-------------------------------|
+| [`dart:html`][dart-html]                <br> HTML elements and other resources for web-based applications. | |
+| [`dart:indexed_db`][dart-indexed_db]    <br> Client-side key-value store with support for indexes. | |
+| [`dart:web_audio`][dart-web_audio]      <br> High-fidelity audio programming in the browser. | |
+| [`dart:web_gl`][dart-web_gl]            <br> 3D programming in the browser. | |
+| [`dart:js`][dart-js]                    <br> _Don't use._ Instead, use the `js` package, as described in [JavaScript interoperability][]. | *DEPRECATED* |
+| [`dart:js_util`][dart-js_util]          <br> _Don't use._ Instead, use the `js` package, as described in [JavaScript interoperability][]. | *DEPRECATED* |
+| [`dart:web_sql`][dart-web_sql]          <br> API for storing data in the browser that can be queried with SQL. | *DEPRECATED* |
```
Comment (kwalrath) [inline src/_guides/libraries/index.md]: The other DEPRECATED libraries have _Don't use._ Should we add that here?

**Comment by kwalrath** [comment]
> I don't love that the notes column across the three tables isn't the same width. Any ideas for how to do that?

Good point. There should be a way to do that in CSS in a way that's OK on both mobile & desktop. Maybe have the first column use 80% of the total width? I can experiment with that...

**Comment by mit-mit** [comment]
> LGTM if my changes LGTY.

Yeah, your changes look great

**Comment by kwalrath** [comment]
I think I've fixed the column thing. ptal @mit-mit

**Comment by kwalrath** [comment]
Staged: https://kw-www-dartlang-1.firebaseapp.com/guides/libraries

**Comment by mit-mit** [comment]
Nice table hack :-)

**Comment by mit-mit** [comment]
The heading 'Libraries that support all Dart platforms' seems a bit long. How about: 'Multi-platform core libraries' ? I like that from a marketing PoV.

**Comment by kwalrath** [comment]
> The heading 'Libraries that support all Dart platforms' seems a bit long. How about: 'Multi-platform core libraries' ? I like that from a marketing PoV.

I used **Multi-platform libraries**, since none of the other headings have "core".

**Pull request merged**